guacamole-common-js/src/main/webapp/modules/AudioRecorder.js

```diff
@@ -183,14 +183,19 @@ Guacamole.RawAudioRecorder = function RawAudioRecorder(stream, mimetype) {
 
     /**
      * A function which directly invokes the browser's implementation of
-     * navigator.getUserMedia() with all provided parameters.
+     * navigator.mediaDevices.getUserMedia() with all provided parameters.
      *
      * @type Function
      */
-    var getUserMedia = (navigator.getUserMedia
-            || navigator.webkitGetUserMedia
-            || navigator.mozGetUserMedia
-            || navigator.msGetUserMedia).bind(navigator);
+
+    if (!navigator.mediaDevices)
```

Comment:
With the old JSDoc comment no longer being valid, there should be some comments here describing what's going on.

Reply:
Added comments.

Reply:
Looks much better, however these new comments are JSDoc comments. JSDoc comments are used for declarations only, and then to document a function, class, variable, etc.:

```javascript
/**
 * This is correct.
 *
 * @private
 */
var foo;
```

They cannot / should not be used for general code which is not a declaration:

```javascript
/**
 * This is not correct.
 */
if (foo) {
    bar();
}
```

Normal `//` comments are the correct thing to use here:

```javascript
// This is correct
if (foo) {
    bar();
}
```

Reply:
Fixed.